src/amo/sagas/guides.js

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+import { call, put, select, takeEvery } from 'redux-saga/effects';
+import { search as searchApi } from 'core/api/search';
+import {
+  FETCH_GUIDES_ADDONS,
+  updateGuideAddonsLoadedStatus,
+} from 'amo/reducers/guides';
+import log from 'core/logger';
+import { createErrorHandler, getState } from 'core/sagas/utils';
+import { loadAddonResults } from 'core/reducers/addons';
+import type { SearchParams } from 'core/api/search';
+import type { FetchGuidesAction } from 'amo/reducers/guides';
+
+export function* fetchGuidesAddons({
+  payload: { errorHandlerId, guid },
+}: FetchGuidesAction): Generator<any, any, any> {
```

Comment:
You can probably use `Saga` from `core/types/sagas`.

Reply:
of course! I looked at one point but needed to rebase and forgot to circle back. I'll do that now..